db/mysql/ops.go

```diff
@@ -49,19 +50,24 @@ func (c *client) SendMessage(ctx context.Context, id string, message *domain.Mes
 	return nil
 }
 
```

Comment:
change the function name of SendMessage to SaveMessage

Reply:
Changed